client.go

```diff
@@ -16,6 +16,7 @@ const (
 	apiEventBase         = "/objects/events"
 	apiLabelBase         = "/objects/labels"
 	apiDeletedBase       = "/objects/deleted"
+	version              = "1.0.0"
```

Comment:
what do you think of moving `version` to config.go? I would prefer to keep this const only for API endpoints.

Reply:
ack. good point!